docs/language/ql-training-rst/slide-snippets/intro-ql-general.rst

```diff
@@ -87,20 +94,17 @@ Find all instances!
 
 .. note::
 
-  The JPL team ran the query across the full Curiosity control software–it identified the original problem, and more than 30 other variants, of which three were in the critical Entry, Descent and Landing module. 
+  The JPL team ran the query across the full Curiosity control software–it identified the original problem, and more than 30 other variants, of which three were in the critical Entry, Descent, and Landing module. 
 
-  The team addressed all issues, and patched the firmware remotely. Not long after, the Curiosity Rover   landed safely on Mars.
+  The team addressed all issues, and patched the firmware remotely. Not long after, the Curiosity Rover landed safely on Mars.
   
   For more detail on the collaboration between Semmle and NASA, see our case study: `Semmle at NASA: Landing Curiosity safely on Mars <https://semmle.com/case-studies/semmle-nasa-landing-curiosity-safely-mars>`__.
 
-.. rst-class:: background2
-
-How it all works
-================
-
 Analysis overview
 =================
 
+.. rst-class:: build
+
 - The database schema is (source) language specific, as are queries and libraries.
 - Multi-language code bases are analyzed one language at a time.
```

Comment:
I don't see these two bullet points. You might need a different approach for this slide.

Reply:
Works for me. Bullets appear _above_ the diagram, which is possibly confusing. I'll swap it round and see how it looks.

Reply:
This is what I see in the Java version of the Introduction, even at full screen size:

![image](https://user-images.githubusercontent.com/1877141/63168731-d53ab800-c02c-11e9-8c5d-384a6a1827c6.png)